spec/seal_spec.rb

```diff
@@ -44,12 +61,28 @@
     context "when in quotes mode" do
       let(:teams) { [lions] }
 
-      it "barks at all teams" do
+      it "posts quotes if a quotes day" do
+        Timecop.freeze(Time.local(2023, 04, 24)) # Monday
         expect(Message).to receive(:new).with("go lions!").and_return(message)
         expect(SlackPoster).to receive(:new).with(lions.channel, anything)
 
         subject.bark(mode: "quotes")
       end
+
+      it "don't bark if not a quotes day" do
```

Comment:
We could simplify things a bit by just having 'lions' as Monday/Tuesday/Wednesday, and not declaring a 'dogs' team. Then we could use the same 'lions' team for both "if a quotes day" and "if not a quotes day" tests.

The small problem with the 'dogs' team test is that we never validate that it _could_ send quotes (or even that the team exists at all). So it would be safer (and shorter) to use one team config to test both scenarios.

Crucially, line 45 is `let(:teams) { [lions] }` so I'm not sure if the 'dogs' and 'cats' teams actually exist when these tests are being run, but the tests pass anyway because you're only checking that they _don't_ send quotes, not that they _do_. Writing a failing test first and then making it pass would avoid this sort of thing.